src/components/Schedule.js

```diff
@@ -34,68 +34,79 @@ const Schedule = () => {
+                                  <p
+                                    className="schedule__location__title"
+                                    title={event.title}
+                                  >
                                     {event.title}
                                   </p>
                                   <a
-                                    className="location__zoom"
+                                    className="schedule__location__zoom"
                                     href={event.location}
+                                    title={event.location}
+                                    // make the link open in a new tab
+                                    target="_blank"
+                                    rel="noopener noreferrer"
                                   >
                                     {event.location}
                                   </a>
                                 </div>
                               </td>
-                              {/* TODO: Need to change hardcoded width here */}
-                              <td width="300px">
-                                <div className="audience">
-                                  <p
-                                    className={
-                                      event.audience ? "audience__type" : ""
-                                    }
-                                  >
-                                    {event.audience}
-                                  </p>
-                                  <p
-                                    className={
-                                      event.company ? "audience__company" : ""
-                                    }
-                                  >
-                                    {event.company}
-                                  </p>
-                                </div>
+                              <td className="schedule__audience">
+                                <p
+                                  className={
+                                    event.audience
+                                      ? "schedule__audience__type"
+                                      : ""
+                                  }
+                                >
+                                  {event.audience}
+                                </p>
+                                <p
+                                  className={
+                                    event.company
+                                      ? "schedule__audience__company"
+                                      : ""
+                                  }
+                                >
+                                  {event.company}
+                                </p>
                               </td>
-                              <td>
+                              <td className="schedule__calendar">
                                 <img src={CalendarIcon} alt="calendar icon" />
```

Comment:
also for the calendar icon we prob want to still show it even if it's on mobile right? Tho this feature is not implemented yet

Reply:
same as above, figured we could put it in expandable row